package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hathor-wallet-headless",
-  "version": "0.22.0-rc1",
+  "version": "0.22.0-rc2",
```

Comment:
Why are you changing this? Shouldn't you change the lib version?

Reply:
Fixed with rebase.

Reply:
See: https://github.com/HathorNetwork/hathor-wallet-headless/blob/5b82047d42409679ce57bafe1547606c37701445/package-lock.json#L1